services/graphite/service.go

```diff
@@ -334,6 +336,14 @@ func (s *Service) handleLine(line string) {
 	// Parse it.
 	point, err := s.parser.Parse(line)
 	if err != nil {
+		switch err := err.(type) {
```

Comment:
This is just a style thing, but with only 1 case, would the code be a little clearer coded as:

```
_, ok := err.(*ErrUnsupportedValue); ok {
    ....
}
```